service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/batchpool/ControlledAzureBatchPoolResource.java

```diff
@@ -266,7 +266,6 @@ public void validate() {
       throw new InconsistentFieldsException("Expected controlled AZURE_BATCH_POOL");
     }
     ResourceValidationUtils.validateAzureBatchPoolId(getId());
-    ResourceValidationUtils.validateAzureVmSize(getVmSize());
```

Comment:
Should we now remove `ResourceValidationUtils.validateAzureVmSize`?

Reply:
The method is also used when creating a VM. I didn’t remove it altogether because for VMs we need to thinking through a replacement using ARM APIs - unlike the batch pool, I don’t think we can assume the VM size is already validated. This should be follow up work.

Reply:
Can you call Azure to do a validation instead of having to maintain a redundant list?